src/components/PomodoroTimer.tsx

```
@@ -194,6 +198,65 @@ const PomodoroTimer: React.FC<PomodoroTimerProps> = React.memo(({ settings }) =>
+            event.preventDefault();
+            handleDoneNext();
+          }
+          break;
+        default:
+          break;
+      }
+    };
+
+    document.addEventListener('keydown', handleKeyDown);
+    return () => document.removeEventListener('keydown', handleKeyDown);
+  }, [handleToggleTimer, resetTimer, handleDoneNext, isActive, phase, event]);
```

Biome (2.4.6)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)